Fluent/Simulation.ipynb

```diff
@@ -55,51 +55,33 @@
     "\n",
     "<br>\n",
     "\n",
-    "**1. Setup Model/s:** (Add the required bullet points with details explanations. Keep the title but remove the color)\n",
-    "- In this problem for flow over a cylinder, the free stream flow is unsteady, or transient. Therefore, the transient model is chosen to develop the CFD simulation: `/define/general/solver/transient`. The critical Reynolds number for flows over cylinder to transition from steady state to unsteady state is about 50. In cases that the Reynolds number, based on cylinder diameter, is less than the critical value of 50 the model will still be viscous, however the steady model should be selected at this step. \n "
```

Comment:
The free stream is uniform and steady. The fact that makes this flow unsteady is going beyond the critical Reynolds number and dominance of the inertial forces within the flow field over the viscous forces. This will lead to flow separation from the cylinder surface and vortex shedding and overall unsteadiness within the flow. Please modify the content according to this explanation.

The explanation of changing to steady model is great. Keep it.